scripts/install_plugins.sh

```diff
@@ -14,19 +14,18 @@ fi
 
 clone() {
 	local plugin="$1"
-	local branch="$2"
-	if [ -n "$branch" ]; then
+	[[ -z "$2" ]] && local branch="" || local branch="--branch $2"
+	if [[ ! $plugin == *"https://"* ]]; then
 		cd "$(tpm_path)" &&
-			GIT_TERMINAL_PROMPT=0 git clone -b "$branch" --single-branch --recursive "$plugin" >/dev/null 2>&1
+			GIT_TERMINAL_PROMPT=0 git clone $branch --single-branch --recursive "https://git::@github.com/$plugin" $plugin >/dev/null 2>&1
```

Comment:
I may be misreading this but does this not break the case where the plugin URL is intended to use SSH? i.e. `git@github.com:user/plugin`, and `git@bitbucket.com:user/plugin`, effectively removing SSH support for non-GitHub sources?